samples/modules/lvgl/accelerometer_chart/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(acceleration_chart)
```

Comment:
Nit: Maybe rename this also.

Reply:
well spotted, thanks! also renamed the test case